middleman-prismic.gemspec

```diff
@@ -1,24 +1,24 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'middleman-prismic/version'
+# -*- encoding: utf-8 -*-
```

Comment:
TIL https://stackoverflow.com/questions/8879237/how-does-the-magic-comment-encoding-utf-8-in-ruby-work